arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,720 @@
+              "$ref": "#/definitions/CommitmentPlan"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_Remove",
+        "description": "Remove an existing Azure ML commitment plan.",
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_Patch",
+        "description": "Patch an existing Azure ML commitment plan resource.",
+        "parameters": [
+          {
+            "name": "patchPayload",
+            "description": "The payload to patch the Azure ML commitment plan with.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommitmentPlan"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommitmentPlan"
+            }
+          }
+        }
+      },
+      "parameters": [
+        {
+         "$ref": "#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "name": "resourceGroupName",
+          "description": "The resource group name.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "name": "planName",
+          "description": "The commitment plan name.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "$ref": "#/parameters/APIVersionParameter"
+        }
+      ]
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearning/commitmentPlans": {
+      "get": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_List",
+        "description": "Retrieve all Azure ML commitment plans in a subscription.",
+        "parameters": [
+          {
+           "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$skipToken",
+            "description": "Continuation token for pagination.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PaginatedResult[CommitmentPlan]"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans": {
+      "get": {
+        "tags": [
+          "CommitmentPlans"
+        ],
+        "operationId": "CommitmentPlans_ListInResourceGroup",
```

Comment:
webservices.json uses ListInResourceGroup... As I'd like to avoid changing that for now, I'd prefer consistency between the two.